### PR TITLE
Css setup

### PR DESCRIPTION
**Pull request by jenn-ah**

### CODE REVIEW

**Comment by jenn-ah** [comment]
ahh just saw the comment about using breaks, i'll take those out

File: src/app/component/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -1,4 +1,22 @@
 <div class="dashboard">
+<<<<<<< HEAD
```
Comment (jaywon) [inline src/app/component/dashboard/dashboard.component.html]: Merge conflict remnants need to be cleaned up.

File: src/app/pages/posts/posts.component.ts [hunk 2]
```diff
@@ -9,39 +9,8 @@ import { SessionService } from '../../services/session.service'
...
-    photo:'',
-    description:'',
-    city:'',
-    state:'',
-    zip_code:null,
-    budget:null,
-    can_bid: false,
-    created_at:'',
-}
+post: any;
```
Comment (jaywon) [inline src/app/pages/posts/posts.component.ts:12]: Believe the post is supposed to have explicit field definitions rather than `any`

**Review by jaywon — CHANGES_REQUESTED** [review]
Please see feedback 😄

File: src/app/pages/posts/posts.component.ts (hunk 2 above)
Comment (jenn-ah) [reply src/app/pages/posts/posts.component.ts:12]: Jen's working on setting up the interfaces file right now and this should be resolved today

File: src/app/component/dashboard/dashboard.component.html (hunk 1 above)
Comment (jenn-ah) [reply src/app/component/dashboard/dashboard.component.html]: Sorry, saw this one after I did the PR.. resolved conflict.

File: server/server.js [hunk 3]
```diff
@@ -48,84 +50,84 @@ passport.serializeUser((username, done) => {
...
-  return res.send(req.user)
-});
+  function (req, res) {
+    return res.send(req.user);
+  });
 
 app.use((req, res, next) => {
-  res.header("Access-Control-Allow-Origin", "*")
-  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
+  res.header("Access-Control-Allow-Origin", "*");
```
Comment (taesup) [inline server/server.js:129]: we really shouldn't be using this at all. The correct way would be to use the proxy.conf.js to set the proxy redirect for angular.

**Review by taesup — COMMENTED** [review]
This one question to be answered before I can approve this.

File: server/server.js (hunk 3 above)
Comment (jenn-ah) [reply server/server.js:129]: I believe Shad added this for the messages component but I'll look into this

**Pull request merged**